src/main/java/jimmy/mcgymmy/model/food/Macronutrient.java

```diff
@@ -43,7 +45,7 @@ public static boolean isValid(String value) {
     }
 
     private boolean isValidAmount(int amount) {
-        return amount >= 0 && amount < 1000;
+        return amount >= LOWER_BOUND && amount < UPPER_BOUND;
```

Comment:
maybe we can change both to inclusive bounds, (so we change the UB to 999). Seems more intuitive/consistent.